platform/frontend/src/components/ui/onboarding-step.tsx

```diff
@@ -0,0 +1,76 @@
+import type { ReactNode } from "react";
+
```

Comment:
Is this a default shadcn component? Do we only add default shadcn components to this folder, @brojd?